setup.py

```diff
@@ -24,14 +24,19 @@ def get_version() -> str:
     name="pygitguardian",
     version=get_version(),
     packages=find_packages(exclude=["tests"]),
-    description="Python Wrapper for GitGuardian's API -- Scan security policy breaks everywhere",
+    description="Python Wrapper for GitGuardian's API -- Scan security "
+    "policy breaks everywhere",
     long_description=read("README.md"),
     long_description_content_type="text/markdown",
     url="https://github.com/GitGuardian/py-gitguardian",
     author="GitGuardian",
     author_email="support@gitguardian.com",
     maintainer="GitGuardian",
-    install_requires=["marshmallow>=3.5", "requests>=2"],
+    install_requires=[
+        "marshmallow>=3.5",
+        "requests>=2",
```

Comment:
While we are at it, I think we should define upper bounds here to ensure we don't get bit by a breaking change. Something like this:

```
"marshmallow>=3.5, <4",
"requests>=2, <3",
```